src/scripts/trigger.js

```diff
@@ -13,16 +14,34 @@ export default class TriggerMesh extends Mesh {
     debug = false;
 
     // Trigger functions - Override these
-    hover;
+    hover() {}
 
-    exit;
+    exit() {};
 
-    select;
+    select() {};
 
-    release;
+    release() {};
 
     functions = {};
 
+    get [Interactions]() {
+      const self = this;
+      return {
```

Comment:
Need to set the `this.isSelected` and `this.isIntersected` booleans.
https://github.com/Group-47-Capstone-2019/Immersive-Web-VR-AR/blob/a7c0fc1e70de8b925e13ac02d1bbb33cf69b7cd0/src/scripts/trigger.js#L30-L40